docs/book/src/migration/v1vsv2.md

```diff
@@ -0,0 +1,66 @@
+# Kubebuilder v1 vs v2
+
+This document cover all breaking changes when migrating from v1 to v2.
+
+The details of all changes (breaking or otherwise) can be found in
+[controller-runtime](https://github.com/kubernetes-sigs/controller-runtime/releases),
+[controller-tools](https://github.com/kubernetes-sigs/controller-tools/releases)
+and [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder/releases)
+release notes.
+
+## Common changes
+
+- All three projects now use `go modules` instead of `dep`. Dep support in
```

Comment:
I think we can simplify this part when it comes to an end-user because they don't care if KB itself uses Go modules or not. We can simply say: 
"v2 project uses Go modules".